app/docs/async-api.yaml

```diff
@@ -0,0 +1,80 @@
+asyncapi: 2.4.0
```

Comment:
Looks a good placeholder/start -- can see you have copied this from other services, which is GOOD -- I immediately picked up the known/common structure for this